src/WpBookingsUiModule.php

```diff
@@ -228,6 +221,20 @@ protected function _getTranslatedStatuses($statuses, $statusesLabels)
         return $translatedStatuses;
     }
 
+    /**
+     * Get list of all services.
+     *
+     * @since [*next-version*]
+     *
+     * @return mixed List of all services.

```

Comment:
Why is the type so wide? Can it return an integer?

Reply:
Got it. I think that `Traversable` is good, can I use it instead?

Reply:
I think you should ask @mecha what he will return. I guess that it will be quite wide. But of course not that wide :D

Reply:
To be honest, its just `array`, because the transformer that I'm using in the event normalizes to array @XedinUnknown So still a pretty vague type.